src/components/Navbar/Navbar/MenuItems.js

```diff
@@ -0,0 +1,22 @@
+export const MenuItems = [
+    {
+        title: 'Daily Grow',
+        url: '#',
+        cName: 'nav-links'
```

Comment:
I don't think cName is a very intuitive name. It's also repeated here. I think we could just add className on the items and map through the other parts since that stays consistent.